src/styles/01-atoms/links/link/_link.scss

```diff
@@ -1,72 +1,59 @@
 /// Link
-$color-link: $nypl-blue-regular;
-$color-link--hover: $nypl-blue-dark;
+$color-link: $nypl-red-regular;
```

Comment:
is there guidance on how to override this?

Reply:
Not currently. This code definitely could use some work but I needed the latest and greatest from Reno for some of the Hero styling. Where do you think that guidance would live? Perhaps in a comment blob at the top of this file?

Reply:
some "Overriding Styles" document in Design System, I think.  The idea is that each product should have its style override sheets

Reply:
I'll create a separate ticket and wiki page for that work, then. It'll be blocked on upcoming color work/`SectionName.tsx` work.

Reply:
I've opened up #107 to capture log this work. Logistically I'm thinking it will happen after any `Template` work upcoming for RN.